### PR TITLE
docs(body size): Added alf body size limit information

### PR DESCRIPTION
**Pull request by therebelrobot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,6 +251,10 @@ For response bodies:
 response.content = { encoding: 'base64', text: 'BASE64_BODY' }
 ```
 
+#### Body sizes
+
+Request/Response bodies sent to the collector MUST be under 10MB. Default for should be set to 2MB. If higher than 10MB for an individual body is needed, please contact support@mashape.com, and we can take a look at your use case with you.
```
Comment (ahmadnassri) [inline README.md:256]: a re-wording might be needed here ... use a list for better clarity.

**Comment by nijikokun** [comment]
:o

**Comment by nijikokun** [comment]
:wave: glad to see you're doing okay, hope you have a good day and a good 🎄 🎁 🎅

**Comment by ahmadnassri** [comment]
:eyes:

**Comment by ahmadnassri** [comment]
:roll_eyes:

**Pull request closed without merging**